docs/howto/writing-an-application-from-scratch/src/main/resources/MyApp.java

```diff
@@ -0,0 +1,36 @@
+package com.example;
```

Comment:
Not sure this file is meant to be here?

Reply:
Of course not, I shouldn't `git add` entire directories. I also shouldn't be working that late. Thank you.